driver-core/src/main/java/com/datastax/driver/core/NettyOptions.java

```diff
@@ -183,22 +183,17 @@ public void afterChannelInitialized(SocketChannel channel) throws Exception {
      * <p>
      * This gives the implementor a chance to close the {@link EventLoopGroup} properly, if required.
      * <p>
-     * The default implementation initiates a {@link EventLoopGroup#shutdownGracefully(long, long, TimeUnit) graceful shutdown}
-     * of the passed {@link EventLoopGroup} instance with no "quiet period" and a timeout of 15 seconds;
-     * then waits uninterruptibly for the shutdown to complete, or the timeout to occur, whichever happens first.
+     * The default implementation initiates a {@link EventLoopGroup#shutdownGracefully() graceful shutdown}
+     * of the passed {@link EventLoopGroup}, then waits uninterruptibly for the shutdown to complete or timeout.
      * <p>
      * Implementation note: if the {@link EventLoopGroup} instance is being shared, or used for other purposes than to
-     * coordinate Netty events for the current cluster, than it should not be shut down here;
+     * coordinate Netty events for the current cluster, then it should not be shut down here;
      * subclasses would have to override this method accordingly to take the appropriate action.
      *
      * @param eventLoopGroup the event loop group used by the cluster being closed
      */
     public void onClusterClose(EventLoopGroup eventLoopGroup) {
-        // shutdownGracefully with default parameters employs a quiet period of 2 seconds
-        // where in pre-Netty4 versions, closing a cluster instance was very quick (milliseconds).
-        // Since we close the channels before shutting down the eventLoopGroup,
-        // it is safe to reduce the quiet period to 0 seconds
-        eventLoopGroup.shutdownGracefully(0, 15, SECONDS).syncUninterruptibly();
+        eventLoopGroup.shutdownGracefully().syncUninterruptibly();
```

Comment:
On another note I noticed this AssertionError during closeDuringClusterInitTest:

```
16:03:40     7709692 [pool-44726-thread-1] INFO  com.datastax.driver.core.Cluster  - New Cassandra host /127.0.1.1:11002 added
16:03:42 Exception in thread "globalEventExecutor-1-2598" java.lang.AssertionError
16:03:42 	at io.netty.util.concurrent.AbstractScheduledEventExecutor.pollScheduledTask(AbstractScheduledEventExecutor.java:83)
16:03:42 	at io.netty.util.concurrent.GlobalEventExecutor.fetchFromScheduledTaskQueue(GlobalEventExecutor.java:110)
16:03:42 	at io.netty.util.concurrent.GlobalEventExecutor.takeTask(GlobalEventExecutor.java:95)
16:03:42 	at io.netty.util.concurrent.GlobalEventExecutor$TaskRunner.run(GlobalEventExecutor.java:226)
16:03:42 	at io.netty.util.concurrent.DefaultThreadFactory$DefaultRunnableDecorator.run(DefaultThreadFactory.java:137)
16:03:42 	at java.lang.Thread.run(Thread.java:701)
```

This caused the test to hang (presumably in Cluster.close, I think?)

I've never seen this before, but looks like it's a known issue (netty/netty#4357) and is fixed in net ty 4.0.33.